lib/components/base/Badge.vue

```diff
@@ -53,16 +53,15 @@ import {
   capitalizeString,
 } from '@'
 
-defineProps({
-  type: {
-    type: String,
-    required: true,
-  },
-  color: {
-    type: String,
-    default: '',
-  },
-})
+withDefaults(
+  defineProps<{
+    type: string
+    color: string
```

Comment:
```suggestion
    color?: string
```

Colour appears to be optional. It also probably should be a union of valid colours, rather than just string (not a part of the suggestion).